environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - codecov
     - coverage[toml]
     - docformatter
+    # - dvc
```

Comment:
Might need this to stay commented out as `conda install dvc=2.0.5` doesn't work yet for Python 3.9 because they don't have the package (see https://anaconda.org/conda-forge/dvc/files?version=2.0.5).

But I think `pip install dvc` works, and they have CI for Python 3.9 as of https://github.com/iterative/dvc/pull/4704. So may need to stick with `pip install` for now.

Reply:
What about adding the following lines to environment.yml?
```
- pip:
  - dvc
```

Reply:
It seems dvc 2.0.6 for Python 3.9 is available on conda-forge now.

Reply:
Yep, I opened up an issue at https://github.com/conda-forge/dvc-feedstock/issues/192 and the dvc team (who seem awesome) added the Python 3.9 package real quick (yay open source)!